src/components/Main/Main.tsx

```diff
@@ -0,0 +1,42 @@
+import { useContext, useMemo, useState } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { Todo } from '../../types/Todo';
+import { TodoList } from '../TodoList/TodoList';
+import { Status } from '../../enums/Status';
+
+export const Main = () => {
+  const { todos, updateTodos } = useContext(TodosContext);
+  const [isToggleAllChecked, setIsToggleAllChecked] = useState(true);
+
+  const isAllChecked = useMemo(() => {
```

Comment:
The condition inside the 'every' method should directly check if the 'completed' property of the todo is true, instead of comparing it with Status.Completed.